.rubocop.yml

```diff
@@ -14,6 +14,7 @@ AllCops:
     - bin/update
     - bin/yarn
     - vendor/bundle/**/*
+    - db/migrate/*
```

Comment:
Migrations are normally written by hand and should follow the standard coding style, please don't exclude them all.